core/errors/rewriter.h

```diff
@@ -15,5 +15,6 @@ constexpr ErrorClass ComputedBySymbol{3509, StrictLevel::False};
 constexpr ErrorClass InitializeReturnType{3510, StrictLevel::False};
 constexpr ErrorClass InvalidStructMember{3511, StrictLevel::False};
 constexpr ErrorClass NilableUntyped{3512, StrictLevel::False};
+constexpr ErrorClass InvalidChainedSig{3513, StrictLevel::False};
```

Comment:
We have added a lint that verifies that all error codes have corresponding documentation in `error-reference.md`, so you'll have to write documentation for this error.

You should be able to reproduce the error by running `tools/scripts/check_error_classes.sh`; if you can't reproduce the error locally, that seems like a bug, and we should fix that separately.

Reply:
Thanks Nathan, CI output made me think it was during `yarn lint`. CI is fixed now.